src/shared_library.c

```diff
@@ -330,6 +343,7 @@ rcutils_get_platform_library_name(
         library_name_platform, strlen(library_name) + 5, "%s.dll", library_name);
     }
   }
+#endif // __MINGW64__
```

Comment:
nit detected by rpr job https://ci.ros2.org/job/ci_linux/21254/testReport/(root)/projectroot/cpplint/

```suggestion
#endif  // __MINGW64__
```

```shell
/home/jenkins-agent/workspace/ci_linux/ws/src/ros2/rcutils/src/shared_library.c:346:  At least two spaces is best between code and comments  [whitespace/comments] [2]
```